package.json

```diff
@@ -49,7 +49,6 @@
     "babel-plugin-object-assign": "1.2.1",
     "babel-plugin-react-transform": "2.0.2",
     "babel-plugin-transform-imports": "2.0.0",
-    "cesium": "1.17.0",
```

Comment:
this depends on how the dependencies in the 

> checkCesium.js

since we are using cesium from within npm dependencies we can get rid of it, since we do not deliver anymore as part of the bundle, plus it was not even used

Reply:
no it is something specific to mapstore2-georchestra